user/tests.py

```diff
@@ -28,3 +31,11 @@ def test_signup_view(self):
             view.func.__name__,
             SignupPageView.as_view().__name__
         )
+
+    def test_signup_form_sucess(self):
+        new_user = User.objects.create_user(self.username, self.email)
```

Comment:
This test is creating the user and then testing if the user it just created is in the database. 
But ideally it should be attempting to fill out the form and then check if by sending the form a user was created in the database.

Reply:
oooh thanks. i will look into it let change the tests